### DISCUSSION TITLE
install from github private packages failed

### DISCUSSION THREAD

**Discussion by Jooyoung-dable** [community/community#118619] [Packages] [answered]
### Select Topic Area

Question

### Body

![스크린샷 2024-04-09 오후 2 15 49](https://github.com/community/community/assets/61184548/cf2fc10e-0333-44b6-a363-47597ddb60bc)

A few days ago in the morning, I encountered this error when installing a module in github private packages. Was there any issue?
No special action was taken, but it was resolved by afternoon.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Jooyoung-dable ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by grandmas-favorite** [comment, marked as answer]
Hi @Jooyoung-dable!

There were no issues that I am aware of - perhaps it was a connectivity issue on your end? Regardless - has the issue remained resolved?

**Comment by Jooyoung-dable** [comment]
Hu @grandmas-favorite! Yes I think it was just temporary problem. I didn't make any special changes but it was resolved over time.
Since there are no issues, I will close this discussion. Thanks!
